app/controllers/courses/vacancies_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Courses
+  class VacanciesController < ApplicationController
+    before_action(
+      :authenticate,
+      :build_course,
+      :build_site_statuses,
+      :build_first_course
+    )
+
+    def edit; end
+
+    def update
+      params.dig(:course, :site_status_attributes)
```

Comment:
Minor thing. If `govuk-lint-ruby` will let you, would it be worth indenting this code here so that it's more obviously a block?